doc/4-instrument/README.md

```diff
@@ -126,14 +126,25 @@ finally, the sequence of values can be directly edited in the text box at the bo
 - in arpeggio macros, a value starting with a `@` is a fixed value as described above.
 - in bitmask-style macros, the values are added up in binary and converted to decimal. see [the hexadecimal guide](../1-intro/hex.md) for more info.
 
-
+in all cases except bitmask macros, right-clicking on the graph opens up an editing menu:

```

Comment:
Graph?

Reply:
Never mind - all good. Doc needs some work.

Reply:
As in "bar graph". Perhaps instead "macro view"? Though to me that means everything involved in the macro, including the relative/fixed bar and such.